roles/download/tasks/sync_file.yml

```diff
@@ -42,12 +32,16 @@
   retries: 4
   delay: "{{ retry_stagger | random + 3 }}"
   when:
-    - download.enabled
-    - download.file
-    - download_run_once
-    - (ansible_os_family not in ["CoreOS", "Container Linux by CoreOS"] and
-      inventory_hostname != download_delegate or
-      download_delegate == "localhost")
+    - ansible_os_family not in ["CoreOS", "Container Linux by CoreOS"]
   tags:
     - upload
     - upgrade
+
+- name: sync_file | Set mode and owner
+  file:
+    path: "{{ download.dest }}"
+    mode: "{{ download.mode | default(omit) }}"
+    owner: "{{ download.owner | default(omit) }}"
+
+- name: "sync_file | Extract file archives"
```

Comment:
nit: no need for double quotes, can you please remove them

Reply:
Done.